common/changes/@microsoft/rush/pr-4167_2023-06-06-09-30.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "fix typo in docs for `startingFolder`",
```

```suggestion
      "comment": "",
```

This won't affect Rush users.